playground/frontend/lib/modules/shortcuts/components/shortcuts_modal.dart

```diff
@@ -18,41 +18,28 @@
 
 import 'package:easy_localization/easy_localization.dart';
 import 'package:flutter/material.dart';
-import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 import 'package:playground/constants/font_weight.dart';
 import 'package:playground/constants/sizes.dart';
 import 'package:playground/modules/shortcuts/components/shortcut_row.dart';
 import 'package:playground/modules/shortcuts/constants/global_shortcuts.dart';
 import 'package:playground_components/playground_components.dart';
 
-const kButtonBorderRadius = 24.0;
-const kButtonWidth = 120.0;
-const kButtonHeight = 40.0;
-const kDialogPadding = 40.0;
+class ShortcutsModalContent extends StatelessWidget {
+  static const _kModalMaxWidth = 400.0;
+  static const _kShortcutsMaxWidth = 200.0;
 
-class ShortcutsModal extends StatelessWidget {
-  final PlaygroundController playgroundController;
-
-  const ShortcutsModal({
+  const ShortcutsModalContent({
+    super.key,
     required this.playgroundController,
   });
 
+  final PlaygroundController playgroundController;
+
   @override
   Widget build(BuildContext context) {
-    AppLocalizations appLocale = AppLocalizations.of(context)!;
-
-    return AlertDialog(
-      title: Text(appLocale.shortcuts),
-      titlePadding: const EdgeInsets.only(
-        top: kDialogPadding,
-        left: kDialogPadding,
-      ),
-      contentPadding: const EdgeInsets.all(kDialogPadding),
-      actionsPadding: const EdgeInsets.only(
-        bottom: kDialogPadding,
-        right: kDialogPadding,
-      ),
-      content: Wrap(
+    return ConstrainedBox(
+      constraints: const BoxConstraints(maxWidth: _kModalMaxWidth),
+      child: Wrap(
```

Comment:
Make vertical, use `spacing`.

Reply:
I got issue connected with flutter rendering, also I didn't got problem we a solving by this fix

Reply:
Use Column, add `alternateWith` or `interleave` function, see https://github.com/dart-lang/collection/issues/227